borg/paperkey.html

```diff
@@ -0,0 +1 @@
+../docs/paperkey.html
```

Comment:
it seems sphinx is easy to get to use paperkey.html from the python package than the other way round. So symlink is not gone. Egg seems to contain the file, but i didn‘t actually test pyinstaller yet.